src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs

```diff
@@ -249,7 +250,12 @@ public void BeforeCommandExecuterExecute([Object]CommandExecuterExecuteBeforeEve
             var context = CreateSmartSqlLocalSegmentContext(eventData.Operation);
             if (eventData.ExecutionContext.Request.RealSql != null)
             {
-                context.Span.AddTag(Common.Tags.DB_STATEMENT, eventData.ExecutionContext.Request.RealSql);
+                //context.Span.AddTag(Common.Tags.DB_STATEMENT, eventData.ExecutionContext.Request.RealSql);
```

Comment:
Why do you comment this line?

Reply:
Below varialbe "sql" include "RealSql"
for example:
ReqlSql="select 1 from T where F1=@F1"
sql="selece 1 from T where F1='aaa'"

Reply:
I mean if you try to delete this, delete this. Git is the time machine. We don't need to keep old codes.

Reply:
Recommit pr after I delete it?

Reply:
Yes, if your plan is to remove this line.

Reply:
done!